packages/cfd/src/Utils/Validator/validator.ts

```diff
@@ -59,16 +104,17 @@ class Validator {
                     return;
                 }
 
-                let is_valid, error_message;
+                let is_valid: boolean;
+                let error_message = '';
```

Comment:
Why did you define a type for `is_valid` but not `error_message` ..?

Reply:
its already declared so no need to assign a string literal else TS will give error